sprokit/processes/core/video_input_process.cxx

```diff
@@ -225,6 +239,7 @@ ::_step()
     push_to_port_using_trait( timestamp, ts );
     push_to_port_using_trait( image, frame );
     push_to_port_using_trait( metadata, metadata );
+    push_to_port_using_trait( frame_rate, d->m_video_reader->frame_rate() );
```

Comment:
Not sure what the use case is for a port with the frame rate? Presumably it is constant?

Reply:
Not necessarily, but in this case it's so that a downsampler after the source can enter a target frame-rate (e.g 1hz)